build/charts/antrea/values.yaml

```diff
@@ -439,3 +439,9 @@ testing:
   coverage: false
   simulator:
     enable: false
+  # -- Don't deploy Antrea on the Nodes labelled with the specific keys.
+  noAntreaNodeWithLabels:
+    enable: false
+    keys:
+      - no-antrea
```

Comment:
This is not the way to do this. This is too specific to be in values.yaml, and in addition it is better to avoid testing-related configurations when possible. There is already a generic `affinity` configuration key, which you can use: https://github.com/antrea-io/antrea/blob/6004f42e153461d2f49c90aafd4389ba28369686/build/charts/antrea/values.yaml#L236

Reply:
Will refine this.

Reply:
Done